src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/exceptions/v2/TaskAlreadyClaimedException.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.exceptions.v2;
+
+import org.zalando.problem.AbstractThrowableProblem;
+import org.zalando.problem.Problem;
+
+import java.net.URI;
+
+import static org.zalando.problem.Status.CONFLICT;
+
+@SuppressWarnings("java:S110")
+public class TaskAlreadyClaimedException extends AbstractThrowableProblem {
+
+    private static final long serialVersionUID = -7314118612753159016L;
+
+    private static final URI TYPE = URI.create("https://github.com/hmcts/wa-task-management-api/problem/task-already-claimed-error");
+    private static final String TITLE = "Task Already Claimed Error";
+    
+    public TaskAlreadyClaimedException(String message, Exception ex) {
```

Comment:
Should add this new problem to the docs.  Here: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1482336467&__ncforminfo=hnDfZ75336XaURs1yHOf7u7qiVTFAMDO8FVkeG4nMeYCuriPfu7BGG2UwvR2GlqLM0ukwbv0dRXr4Fp_SIuFdROVn6VChbl7

Reply:
line 24 added to related document, please check